tapchannel/aux_sweeper.go

```diff
@@ -807,6 +799,20 @@ func importOutputProofs(scid lnwire.ShortChannelID,
 		log.Infof("Fetching funding input proof, locator=%v",
 			spew.Sdump(inputProofLocator))
 
+		// First, we'll make a courier to use in fetching the proofs we
+		// need.
+		proofFetcher, err := proofDispatch.NewCourier(
+			courierAddr, proof.Recipient{
+				ScriptKey: scriptKey,
+				AssetID:   proofPrevID.ID,
+				Amount:    proofToImport.Asset.Amount,
+			},
+		)
+		if err != nil {
+			return fmt.Errorf("unable to create proof courier: %w",
+				err)
+		}
```

Comment:
While we're at it, we should add a `defer proofFetcher.Close()` here. Once a loop iteration completes, we no longer need the gRPC/courier connection.